src/guide/typescript/overview.md

```diff
@@ -175,10 +175,6 @@ const count = ref(1)
 </template>
 ```
 
-See also:
-
-- [TypeScript-only Features in `<script setup>`](/api/sfc-script-setup.html#typescript-only-features)
```

Comment:
I think this link can be replaced by "Using `<script setup>` (with TypeScript)" section: https://vuejs.org/guide/typescript/composition-api.html#using-script-setup

What do you think? :-)

Reply:
@Jinjiang Thank you for your review!

"TypeScript-only Features in `<script setup>`" is now not only props.
I don't think it's necessary to put links to all the features here, so it's okay to delete this.

Reply:
Yeah, fair enough. Thanks for the explanation.